elasticsearch.md

```diff
@@ -1111,6 +1111,34 @@ This authentication and authorization connector represents the secure channel \(
 
 Continue reading about this in the kibana plugin documentation, in the dedicated [SAML section](kibana.md#saml)
 
+### Protecting ROR internals
+
+#### `ror_internal_api`
+
+`ror_internal_api: "allow" # or "forbid"`
+
+ROR plugin has an internal API for an in-index settings management. The settings are stored in `.readonlyrest` index (or [custom one](./kibana.md#configuration)). Moreover, when an audit collector is enabled, audit entries will be stored in the audit index (by default in the index defined by a template `readonlyrest_audit-'yyyy-MM-dd`). All of these indices and the API should be protected and probably only an admin or a superuser should be able to have an access to it. To allow access to the ROR internal `ror_internal_api: "allow"` should be added to the admin/superuser's block. When `ror_internal_api` is not used in a given block, ROR implicitly adds `ror_internal_api: "forbid"` step which will be checked after block is matched.
```

Comment:
the audit indices are not going to be inaccessible (for reads) by default , right? I mean, makes sense to make them not writeable directly by default. But if I want to allow read to them, I don't want to give implicitly the permission to delete/change ".readonlyrest".

Reply:
the new rule forbid an access when:
* it's NOT a readonly request to one of ROR indices (audit, in-index settings) OR
* it's a ROR config action (new API or old API)

so, yes - you don't have to add explicitly a permission to read eg. .readonlyrest index, but you cannot write to it without 
adding `ror_internal_api: "allow"` to the block.